tests/ltp/patches/chown04.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/testcases/kernel/syscalls/chown/chown04.c b/testcases/kernel/syscalls/chown/chown04.c
+index 1f3ed412b..487aec16e 100644
+--- a/testcases/kernel/syscalls/chown/chown04.c
++++ b/testcases/kernel/syscalls/chown/chown04.c
+@@ -37,7 +37,8 @@
+  *   6) chown(2) returns -1 and sets errno to ENOENT if the specified file
+  *              does not exists.
+  */
+-
++/* Patch Description: Test cases were failing as there was not sufficient memory to use for test image creation for a loop device for test purpose.
```

Comment:
Can you keep lines to a reasonable length? Most people don't have linewrapping enabled in their editors and this makes it annoying to read, also on GitHub.